README.md

```diff
@@ -15,35 +17,22 @@
 
 ## Developing
 
-This project is built on top of [Citizen Rails](https://github.com/citizencode/citizen-rails).
-
-**See [README-citizen-rails-template.md](README-citizen-rails-template.md) for instructions on
-using the Citizen Rails template project as a base for your own project.**
-
 ### Getting Started
 
 * After cloning this repo, run: `bin/setup`
-* Run `bundle exec citizen doctor` repeatedly until all the failures are fixed.
 * Read through the [CONTRIBUTING.md](CONTRIBUTING.md) file to learn how this
   team operates.
 
-### citizen-scripts
-
-The Gemfile includes `citizen-scripts` which is a collection of useful dev utils. It includes a command
-called `citizen`. Running `citizen` with no arguments will show usage information. Depending on your
-Ruby setup, you might need to run `bundle exec citizen` instead of just `citizen`.
-
 ### Day-to-day
 
 * Run the server: `heroku local` and [http://localhost:3000](http://localhost:3000)
-* Run tests: `bundle exec citizen test`
-* Pull, run tests, push: `bundle exec citizen pushit`
+* Run tests and Rubocop: `rake`
 
 ### Deploying
 
 * [CircleCI](https://circleci.com/gh/codeforamerica/michigan-benefits) is currently set up to
   deploy green builds to **staging**.
-* Use `citizen promote` to promote from staging to **production**. If you want to promote
+* Use heroku pipelines to promote from staging to **production**. If you want to promote
```

Comment:
What's the exact command we would run to do this?

Looks like:
`heroku pipelines:promote -r staging`

https://devcenter.heroku.com/articles/pipelines#promoting